ui/app/routes/jobs/job.js

```diff
@@ -15,7 +15,17 @@ export default class JobRoute extends Route {
   }
 
   model(params) {
-    const [name, namespace = 'default'] = params.job_name.split('@');
+    let name,
+      namespace = 'default';
+    const { job_name } = params;
+    const delimiter = job_name.lastIndexOf('@');
+    if (delimiter !== -1) {
+      name = job_name.slice(0, delimiter);
+      namespace = job_name.slice(delimiter + 1);
+    } else {
+      name = job_name;
+      namespace = 'default';
```

Comment:
Not a blocker, but do you need this again?

Reply:
Good catch!